### ISSUE TITLE
Re-order Saved Views

### ISSUE THREAD

**Issue by argash** [jonaswinkler/paperless-ng#368]
It's quite possible I am missing something, but is there a way to re-order saved views?

**Comment by jonaswinkler** [comment]
No, you did not miss anything, they are sorted alphabetically. Should paperless have some means to reorder them manually?

**Comment by argash** [comment]
YES! For example I always want Inbox and TODO at the top Then I'd like to have Current Year and Previous Year, then my house followed by the kids then the cars
